README.md

```diff
@@ -465,6 +465,35 @@ defmodule Dispatcher do
   end
 end
 ```
+### Fix slow start times
+
+If you are experiencing slow start times (5-10 minutes) with Elixir services such as mu-dispatcher. on recent kernels, you may need to limit file descriptors via `ulimit`. Follow these steps to decrease the startup time:
+
+1. **Edit Docker Daemon Configuration**:
+   
+   Open the Docker daemon configuration file located at `/etc/docker/daemon.json` and add the following configuration:
+
+   ```json
+   {
+     "default-ulimits": {
+       "nofile": {
+          "Name": "nofile",
+          "Soft": 104583,
+          "Hard": 104583
+       }
+     }
+   }
+   ```
+
+2. **Restart Docker**:
+   
+   After updating the configuration, restart the Docker service to apply the changes:
+
+   ```sh
+   sudo systemctl restart docker
+   ```
+
+By setting the `nofile` ulimit, you can significantly reduce the startup time for your Elixir services.
```

Comment:
This line feels a bit redundant and can be removed since it's already mentioned [here](https://github.com/mu-semtech/mu-dispatcher/pull/14/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R470) that this setting will improve boot times.